documentation/docs/verifiable_credentials/revoke.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Verifiable Credential Revocation
+sidebar_label: Revocation
+description: Explain how a VC is revoked.
+image: /img/Identity_icon.png
+keywords:
+- verifiable
+- credentials
+- revoke
+- Merkle Key Collections
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+import revoke_vc_js from  '!!raw-loader!./../../../bindings/wasm/examples/src/revoke_vc.js';
+import revoke_vc_rs from  '!!raw-loader!./../../../examples/low-level-api/revoke_vc.rs';
+
+TODO: Explain how Verifiable Credentials are revoked using Merkle Key Collections
+
+## Example
+
+This example shows how you can revoke a [Verifiable Credential(VC)](overview) by removing a verification method (public key) from the DID Document of the Issuer. This means the VC can no longer be validated. This would invalidate every VC signed with the same public key, meaning the issuer would have to sign every VC with a different key.
```

Comment:
```suggestion
This example shows how you can revoke a [Verifiable Credential(VC)](overview) by removing a verification method (public key) from the DID Document of the Issuer. This means the VC can no longer be validated. This would invalidate every VC signed with the same public key, meaning the Issuer would have to sign every VC with a different key.
```